packages/spelunker-web/src/components/core/Box/Tabbed/Tab.jsx

```diff
@@ -1,10 +1,6 @@
 import React from 'react';
 
 const Tab = (props) => {
-  if (!props.active) {
-    return null;
-  }
-
   const Component = props.component;
   return <Component {...props} />;
 };
```

Comment:
I know this isn't doing much anymore, but I thought it'd be good to keep around as an abstraction, and to potentially filter out tab-specific props (like `path` and `label`) in the future.

Reply:
👍